src/components/header/Header.jsx

```diff
@@ -0,0 +1,93 @@
+import { useState, useEffect } from 'react';
+const Header = () => {
+    const [isNavOpen, setIsNavOpen] = useState(false);
+
+    const toggleNav = () => {
+        setIsNavOpen(!isNavOpen);
+    };
+
+    useEffect(() => {
+        const checkScreenSize = () => {
+            if (window.innerWidth >= 1024) {
+                setIsNavOpen(false);
+            }
+        };
+
+        window.addEventListener('resize', checkScreenSize);
+
+        checkScreenSize();
+
+        return () => window.removeEventListener('resize', checkScreenSize);
+    }, []);
+    return (
+        <header className="px-4 opacity-100 backdrop-blur-18 relative ">
+            <div className="container mx-auto p-6 flex justify-between items-center">
+                <a href="/" className="flex gap-4 items-center">
+                    <div className="w-[17px] h-6 md:w-6 md:h-8 relative z-10">
+                        <img className="w-full h-full" src="./logo__1.png" alt="Logo 1" />
+                    </div>
+                    <div className="w-36 h-5 md:h-6 md:w-[182px] relative z-10">
+                        <img className="w-full h-full" src="./logo__2.png" alt="Logo 2" />
+                    </div>
+                </a>
+                <nav
+                    className={`lg:flex z-] ${
+                        isNavOpen
+                            ? 'flex flex-col items-start absolute top-6 bg-white h-screen'
+                            : 'hidden gap-12 items-center justify-between'
+                    }`}>
+                    <ul className="flex gap-12 lg:flex-row flex-col mt-[232px] lg:mt-0 font-sans nav-items">
+                        <li className="text-[28px]  text-[#828796] font-semibold lg:font-normal hover:lg:font-semibold lg:text-[16px] hover:text-[#2f3750]">
```

Comment:
don't hardcode the value [] add this to the config or use the nearest default value

Reply:
for text size, color etc